### ISSUE TITLE
still maintained?

### ISSUE THREAD

**Issue by IzzySoft** [benkaiser/stretto-android-sync#5]
As there hasn't been a commit for more than 2 years: is this project still maintained (and are updates to be expected)?

**Comment by benkaiser** [comment]
I'm doing bugfixes as they arise, but for the most part the codebase has continued to work for me. Have you run into a specific issue @IzzySoft ?

The guidance generally is that there is a new version of Stretto, hosted on the next branch: 
https://github.com/benkaiser/stretto/tree/next
And accessible via the web here:
https://next.kaiserapps.com/
You can import an existing Stretto library into the new version of Stretto.

There is also a mobile app in react-native, but I haven't built a formal release version of it yet:
https://github.com/benkaiser/stretto-mobile-next

So essentially in the short to medium term, stretto-android-sync will go away. But stretto-next will take it's place.

**Comment by IzzySoft** [comment]
No issues except that your app was not updated for quite a while in [my repo](https://apt.izzysoft.de/fdroid/repo). I just checked all of its 430+ apps if they're still alive – don't want the users run into disappointment by installing an app that's no longer maintained (unless they know the fact in advance).

So thanks for your response! I count that as "still alive, but the next (stable) release might not come that soon". If your "next" branch uses a different package name, and you want "easy maintenance" for your testers by adding that to my repo as well, just let me know!

**Comment by benkaiser** [comment]
I see. What is that repo used for?

I've updated the README to clarify the license for this project. I would prefer people not distributing the application except via the sources provided (Github and Play Store), however you may do so if you insist.

The next release of the app will come under a new package name, and I will likely follow the same course of not wishing it to be distributed outside of Github and the Play Store, as it is the small way in which I am able to receive revenue from users by downloading through the Play Store.

**Comment by IzzySoft** [comment]
Guess you know Debian? They have a repo called "nonfree". That's what my repo is for [F-Droid](https://f-droid.org/): Apps that cannot fully meet the strong inclusion criteria still can get into my repo. Often they then work on becoming fully compliant, and finally move into the official repo (if you want your app there, just let me know and I gladly assist you). Sometimes they get "worse" (e.g. include 5 new trackers), and get kicked out of my repo as well (ouch). Mostly they stay as they are, and stay in my repo.

Advantage of that? For people like me who either avoid Playstore or cannot access it, this offers a central place with additional apps which, once installed, will be automatically updated on their devices when an update is released. As I don't have "analytics" (being a privacy advocate, that would be a no-go) I cannot say how many people are using y repo. But from the raw webstats, its about 150k visitors a month.

So basically, I pick up the APK files from such Github repos providing them – and offer my users easier access to (and easier finding of) them. If you really don't want that for your app, I of course respect it and remove it from my repo – though I'd be sorry to do so and thus hope you can live with it.
